lib/models.py

```diff
@@ -1,8 +1,8 @@
-from datetime import datetime
+from datetime import datetime, timedelta
 
-from . import output, cache
+from . import output, cache, api
 
-from settings import OUTPUT_DATE_FORMAT
+from settings import OUTPUT_DATE_FORMAT, TIME_OFFSET
```

Comment:
Could you please wrap this in a try - catch ... It would really be awkward if user's got errors after updating to the newest version.
```
try: 
    from settings import TIME_OFFSET
except:
    TIME_OFFSET = 0
```  

I know that this settings system is really poorly designed. I hope to fix it soon or to merge the refactoring you mentioned in the PR.